client/dashboard/sites/site-restore-modal/content-info.tsx

```diff
@@ -0,0 +1,74 @@
+import { useMutation } from '@tanstack/react-query';
+import {
+	__experimentalHStack as HStack,
+	__experimentalText as Text,
+	__experimentalVStack as VStack,
+	Button,
+} from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { createInterpolateElement } from '@wordpress/element';
+import { sprintf, __ } from '@wordpress/i18n';
+import { store as noticesStore } from '@wordpress/notices';
+import { siteRestoreMutation } from '../../app/queries/site';
+import type { Site } from '../../data/types';
+
+interface ContentInfoProps {
+	site: Site;
+	onClose: () => void;
+}
+
+export default function SiteRestoreContentInfo( { site, onClose }: ContentInfoProps ) {
+	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
+	const mutation = useMutation( siteRestoreMutation( site.ID ) );
+
+	const handleSubmit = ( e: React.FormEvent ) => {
+		e.preventDefault();
+
+		mutation.mutate( undefined, {
+			onSuccess: () => {
+				createSuccessNotice(
+					/* translators: %s: site domain */
+					sprintf( __( '%s has been restored.' ), site.slug ),
```

Comment:
I got this when restoring a deleted site. 🙈 

<img width="276" alt="image" src="https://github.com/user-attachments/assets/df552c9a-010a-46a2-989d-25c498bc08b9" />

Reply:
oops...